make.config

```diff
@@ -13,20 +13,20 @@ INSTALLDIR := $(ROOT)/_install
 ################################################################################
 
 # Whether the DFTD3 library (dispersion) should be linked.
-WITH_DFTD3 := 1
+WITH_DFTD3 := 0
 
 # Whether DFTD3 should be compiled during the build process. If set to 1 (yes),
 # you will have to download the dftd3 library before starting the build using
 # the utils/get_opt_externals tool.
-COMPILE_DFTD3 := 1
+COMPILE_DFTD3 := 0
 
 # Set the compile time include and the link time library options for
 # dftd3-lib. Ignored if WITH_DFTD3 has been disabled or COMPILE_DFTD3 enabled.
 DFTD3_INCS := -I/usr/local/include/dftd3-lib
 DFTD3_LIBS := -L/usr/local/lib -ldftd3
 
 # Whether the ARPACK library (needed by TD-DFTB) should be linked with DFTB+
-WITH_ARPACK := 1
+WITH_ARPACK := 0
```

Comment:
Please set those variables back to their original values. (You should override them in your local make.arch file, rather then in the official make.config)